cmd/funcbench/main.go

```diff
@@ -175,6 +175,16 @@ func main() {
 			data, err = exec.Command("git", "checkout", "--", path).CombinedOutput()
 			if err != nil {
 				log.Println(string(data))
+
+				// New file appeared and is not tracked, then it can be removed
+				if strings.Contains(string(data), "did not match any file") {
+					data, err = exec.Command("rm", path).CombinedOutput()
+					if err != nil {
+						log.Println(string(data))
+						log.Fatalln(err)
```

Comment:
`Fatalln` and `Println` can take more than a single argument like
`log.Println("err:", err,"data:", string(data))`